internal/tofu/eval_import.go

```diff
@@ -185,6 +186,7 @@ func parseImportIndexKeyExpr(evalCtx EvalContext, expr hcl.Expression, keyData i
 		return idx, diags
 	}
 
+	// TODO ephemeral - we might need to add ephemeral mark too
```

Comment:
Yep, these instance keys can't be ephemeral

Reply:
Solved in 864420b21980d79510c3c1fc2b329389c879f23d